### PR TITLE
Dev 2532

### PR DESCRIPTION
**Pull request by charlesshale**

### CODE REVIEW

File: cluster/src/test/java/com/hartwig/pipeline/testsupport/TestInputs.java [hunk 1]
```diff
@@ -224,20 +228,18 @@ public static SageOutput sageSomaticOutput() {
...
 
     public static PaveOutput paveGermlineOutput() {
         return PaveOutput.builder(PaveGermline.NAMESPACE)
                 .status(PipelineStatus.SUCCESS)
-                .maybeFinalVcf(gsLocation(somaticBucket(PaveGermline.NAMESPACE),
-                        RESULTS + TUMOR_SAMPLE + ".germline." +
-                                // String.format(".%s.%s.", SageGermlinePostProcess.SAGE_GERMLINE_FILTERED, PAVE_FILE_ID) +
-                                FileTypes.GZIPPED_VCF))
+                .maybeFinalVcf(gsLocation(somaticBucket(PaveGermline.NAMESPACE), RESULTS + TUMOR_SAMPLE + ".germline." +
+                        // String.format(".%s.%s.", SageGermlinePostProcess.SAGE_GERMLINE_FILTERED, PAVE_FILE_ID) +
```
Comment (nedleitch) [inline cluster/src/test/java/com/hartwig/pipeline/testsupport/TestInputs.java:241]: Delete?

File: cluster/src/test/java/com/hartwig/pipeline/testsupport/TestInputs.java [hunk 2]
```diff
@@ -224,20 +228,18 @@ public static SageOutput sageSomaticOutput() {
     public static PaveOutput paveSomaticOutput() {
         return PaveOutput.builder(PaveSomatic.NAMESPACE)
                 .status(PipelineStatus.SUCCESS)
-                .maybeFinalVcf(gsLocation(somaticBucket(PaveSomatic.NAMESPACE),
-                        RESULTS + TUMOR_SAMPLE + ".somatic." +
-                                // String.format(".%s.%s.", SageSomaticPostProcess.SAGE_SOMATIC_FILTERED, PAVE_FILE_ID) +
-                                FileTypes.GZIPPED_VCF))
+                .maybeFinalVcf(gsLocation(somaticBucket(PaveSomatic.NAMESPACE), RESULTS + TUMOR_SAMPLE + ".somatic." +
+                        // String.format(".%s.%s.", SageSomaticPostProcess.SAGE_SOMATIC_FILTERED, PAVE_FILE_ID) +
```
Comment (nedleitch) [inline cluster/src/test/java/com/hartwig/pipeline/testsupport/TestInputs.java:232]: Delete?

File: cluster/images/create_public_image_local.sh [hunk 3]
```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+LOCATION="europe-west4"
+ZONE="${LOCATION}-a"
+IMAGE_PROJECT="hmf-pipeline-development"
+which jq >/dev/null || (echo Please install jq && exit 1)
```
[the remark this answers is not in the document]
Comment (charlesshale) [reply cluster/images/create_public_image_local.sh]: I thought we may want to keep that one for using local JARs?

File: cluster/src/test/java/com/hartwig/pipeline/testsupport/TestInputs.java (hunk 2 above)
Comment (charlesshale) [reply cluster/src/test/java/com/hartwig/pipeline/testsupport/TestInputs.java:232]: just the commented line? yes, thanks

File: cluster/src/test/java/com/hartwig/pipeline/testsupport/TestInputs.java (hunk 1 above)
Comment (charlesshale) [reply cluster/src/test/java/com/hartwig/pipeline/testsupport/TestInputs.java:241]: yes as above, thanks

**Pull request merged**